src/main/java/com/powsybl/openloadflow/equations/EquationSystem.java

```diff
@@ -67,6 +59,26 @@ private void update() {
             invalide = false;
         }
 
+        private void reIndex() {
```

Comment:
Just a question for later: all the AS will work around this function "reindex()" ? I mean we will update the equation system at this level and not at a lower level (in the admittance matrix) ?

Reply:
Exactly.